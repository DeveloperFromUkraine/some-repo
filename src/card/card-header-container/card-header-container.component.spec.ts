import { ComponentFixture, TestBed } from '@angular/core/testing';
import { CardHeaderContainerComponent } from './index';
import { ComponentTest } from '../../../test/test-bed/component';

describe('Card Header Container', () => {
  let fixture: ComponentFixture<CardHeaderContainerComponent>;

  beforeEach(async () => {
    await ComponentTest.createTestBed([], [CardHeaderContainerComponent]);
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(CardHeaderContainerComponent);
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
