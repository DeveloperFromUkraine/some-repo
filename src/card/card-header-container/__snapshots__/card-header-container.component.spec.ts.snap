// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Header Container should match snapshot 1`] = `
<ign-card-header-container>
  <div
    _ngcontent-c0=""
    class="card-header-container"
    role="heading"
  />
</ign-card-header-container>
`;
